projects/plugins/jetpack/_inc/lib/admin-pages/class.jetpack-admin-page.php

```diff
@@ -51,8 +67,10 @@ public function on_jetpack_loaded( $jetpack ) {
 			Identity_Crisis::validate_sync_error_idc_option() && ! Jetpack_Options::get_option( 'safe_mode_confirmed' )
 		);
 	}
-
-	function add_actions() {
+	/**
```

Comment:
It seems clearer when reading the file if there's a blank line between the end of the previous method and the next method's doc block.
```suggestion

	/**
```